src/samplers/gamma.jl

```diff
@@ -6,22 +6,22 @@
 
 # suitable for shape >= 1.0
 
-struct GammaGDSampler <: Sampleable{Univariate,Continuous}
-    a::Float64
-    s2::Float64
-    s::Float64
-    i2s::Float64
-    d::Float64
-    q0::Float64
-    b::Float64
-    σ::Float64
-    c::Float64
-    scale::Float64
+struct GammaGDSampler{T} <: Sampleable{Univariate,Continuous}
```

Comment:
Shouldn't `T` be constrained to `Number` or `Real`?

Reply:
Indeed! I correct it.